flow/directives.py

```diff
@@ -255,32 +255,36 @@ def _evaluate(value, jobs):
 
 
 class _OnlyType:
```

Comment:
Since this class now accept multiple types, `_OnlyTypes` would be a more appropriate name.